python/pyarrow/_parquet.pxd

```diff
@@ -557,6 +557,7 @@ cdef extern from "parquet/arrow/writer.h" namespace "parquet::arrow" nogil:
 
         CStatus WriteTable(const CTable& table, int64_t chunk_size)
         CStatus NewRowGroup(int64_t chunk_size)
```

Comment:
@pitrou can we remove the previous signature without deprecation? I assume the pxd files are not widely used.